_config.yml

```diff
@@ -1,3 +1,4 @@
 remote_theme: "owasp/www--site-theme"
 plugins:
- - jekyll-include-cache-0.2.0
\ No newline at end of file
+ - jekyll-include-cache-0.2.0
+  - jekyll-redirect-from
```

Comment:
```suggestion
 - jekyll-include-cache-0.2.0
 - jekyll-redirect-from
 
```

Reply:
One space? That's weird. Shouldn't they both be 2 spaces instead?

Reply:
I was just going for consistency and the file ending with a blank line 👍